### ISSUE TITLE
Modify feature geometry at runtime

### ISSUE THREAD

**Issue by DiabloRusso** [pauldendulk/Mapsui#111]
Hi!
How to modify feature geometry at runtime, for example, I get the coordinates in real time, and I want to draw a trajectory?
I add a point to feature geometry and update layer:
`var line = _feature.Geometry as LineString;
if (line != null)
    {
      line.Vertices.Add(point);
      _layer.Update();
    }`
My updatable layer:
`public class UpdatableLayer : Layer
    {
        public void Update()
        {
            OnDataChanged(new DataChangedEventArgs());
        }
    }`
Thanx, DiabloRusso

**Comment by pauldendulk** [comment]
This scenario did not get any attention so far but should work properly. At the moment it is possible with the workaround used in the sample I just committed: https://github.com/pauldendulk/Mapsui/blob/master/Samples/Mapsui.Samples.Common/Maps/MutatingTriangleSample.cs

There are two issues I needed to a workaround for:
1. Caching of RenderedGeometry. This can be cleared on the feature. In your sample: ```_feature.RenderedGeometry.Clear();```
2. Trigger the DataChanged event. In this case through the MemoryProvider ViewChanged method. But this could be different in other providers.
